explain the difference between bootnodes and static nodes (#1732)
hyperledger/besu-docs: docs/private-networks/how-to/configure/bootnodes.md

Bootnodes and static nodes are parallel methods for finding peers. Depending on your use case, you can use only bootnodes, only static nodes, or both bootnodes and static nodes.

> Besu attempts connection to all bootnodes once, at startup. Static nodes are different in that reconnection is attempted periodically if the connection fails or is lost.

Reviewer: ```suggestion
When connecting to bootnodes, Besu attempts to connect to all bootnodes at once, at startup.
When connecting to static nodes, Besu attempts to reconnect periodically, if the connection fails or is lost.
```
